models/demos/tg/llama3_70b/tt/llama_common.py

```diff
@@ -82,3 +82,44 @@ def tt_sharded_all_gather(input_tensor, mesh_device, cluster_axis, dim, num_link
         mesh_device=mesh_device,
         memory_config=memory_config,
     )
+
+
+def tt_sharded_distributed_rmsnorm(mesh_device, inp, epsilon, gamma):
+    core_grid = (4, 8)
+    num_cores = core_grid[0] * core_grid[1]
+    input_sharded_memory_config = ttnn.create_sharded_memory_config(
+        shape=(1, 1, 32, 2048),
+        core_grid=ttnn.CoreGrid(y=core_grid[0], x=core_grid[1]),
+        strategy=ttnn.ShardStrategy.WIDTH,
+    )
+    sharded_program_config = ttnn.LayerNormShardedMultiCoreProgramConfig(
+        compute_with_storage_grid_size=(core_grid[1], core_grid[0]),
+        subblock_w=(2048 // num_cores) // 32,
```

Comment:
Can we parametrize 2048? Maybe we start reusing same method for some other model and we definitely don't want to rely on hard coded value.